source/libphone/libphone.cpp

```diff
@@ -0,0 +1,111 @@
+#include "libphone.hpp"
+#include "utils/utils.hpp"
+
+namespace LibPhone {
+Phone::Phone(zenohc::Session& session)
+	: m_app(uWSAppWrapper({.passphrase = "1234"})),
```

Comment:
Whose passphrase is this?

Reply:
I think it's Rick from Rick&Morty, will remove it in the last stable commit